packages/rs-dpp/src/reduced_platform_state/v0/mod.rs

```diff
@@ -0,0 +1,25 @@
+use crate::block::block_info::BlockInfo;
+use crate::block::extended_block_info::ExtendedBlockInfo;
+use crate::fee::default_costs::EpochIndexFeeVersionsForStorage;
+use crate::util::deserializer::ProtocolVersion;
+use bincode::{Decode, Encode};
+use platform_value::Bytes32;
+
+/// Reduced Platform State for Saving V0
+#[derive(Clone, Debug, Encode, Decode)]
+pub struct ReducedPlatformStateForSavingV0 {
+    /// Information about the genesis block
+    pub genesis_block_info: Option<BlockInfo>,
```

Comment:
We set it to None every block (not sure why) so I guess you don't need it then https://github.com/dashpay/platform/blob/2fdd94994d3a550179046fad701bf78229d36834/packages/rs-drive-abci/src/execution/platform_events/block_end/update_state_cache/v0/mod.rs#L56